packages/core/src/llm/executor/stream-processor.ts

```
@@ -0,0 +1,339 @@
+                        });
+
+                        this.eventBus.emit('llm:tool-call', {
+                            toolName: event.toolName,
+                            args: event.input as Record<string, unknown>,
+                            callId: event.toolCallId,
+                        });
+                        break;
+
+                    case 'tool-result':
+                        // PERSISTENCE HAPPENS HERE
+                        const rawResult = event.output;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)